portfolio/src/main/webapp/script.js

```diff
@@ -25,31 +27,38 @@ $(document).ready(function () {
 });
 
 const animateScrollTop = (targetClass) => {
-  $("html").animate(
-    {
+  $("html").animate({
       scrollTop: $("." + targetClass).offset().top,
     },
     2000
```

Comment:
Can you update prettier to use trailing commas.

Reply:
What do you mean by this?

Reply:
https://prettier.io/docs/en/options.html#trailing-commas

I'm all for trailing commas on objects and arrays, but I vote against trailing comma for a function argument. The default makes sense to me.